docs/pages/reference/backends.mdx

```diff
@@ -585,6 +592,13 @@ To enable these options you will need to update the IAM Policy for Teleport.
 }
 ```
 
+<Admonition
+  type="warning"
+  title="on_demand with autoscaling"
+>
+  If `on_demand` is set to true, auto scaling will be disabled and read/write capacities will be set to zero
```

Comment:
```suggestion
  If `on_demand` is set to true, the settings for read/write capacity and auto scaling will be ignored.
```
I don't think we actually mention the `read_capacity_units` and `write_capacity_units` tho, should this only mention autoscaling?